packages/create-package/template/test/pages/index.html

```diff
@@ -1,59 +1,53 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
 	<meta charset="utf-8">
-
-	<title>{{INIT_PACKAGE_VAR_TAG}}</title>
+	<title>UI5 Web Components</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta charset="utf-8">
-
-	<script data-ui5-config type="application/json">
-		{
-			"theme": "sap_horizon_dark",
-			"language": "EN"
-		}
-	</script>
-
 	<link rel="stylesheet" type="text/css" href="./css/index.css">
-	<script src="../../bundle.esm.js" type="module"></script>
+	<script src="../../dist/bundle.esm.js" type="module"></script>
```

Comment:
bundle should not be used from dist. Use vite env `%VITE_BUNDLE_PATH%` variable added from https://github.com/SAP/ui5-webcomponents/pull/11420/